### PR TITLE
sndpeek: init at 1.4

### PR DESCRIPTION
**Pull request by laikq**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2569,6 +2569,11 @@
     github = "kylewlacy";
     name = "Kyle Lacy";
   };
+  laikq = {
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Please split this into a separate commit.

**Comment by mmahut** [comment]
Init packages should always go to master first.

**Comment by laikq** [comment]
Ahoy, I hope this does the trick -- I'm very much a newbie, this is my first pull request to a larger project and I hope I can contribute something and learn more about how this project works.

File: pkgs/applications/audio/sndpeek/default.nix [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchurl, libsndfile, freeglut, alsaLib, mesa, libGLU, libX11, libXmu
+, libXext, libXi }:
+
+stdenv.mkDerivation rec {
+  pname = "sndpeek";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://soundlab.cs.princeton.edu/software/sndpeek/files/sndpeek-1.4.tgz";
```
Comment (Lassulus) [inline pkgs/applications/audio/sndpeek/default.nix]:
```suggestion
    url = "https://soundlab.cs.princeton.edu/software/sndpeek/files/sndpeek-${version}.tgz";
```

Comment (laikq) [reply pkgs/applications/audio/sndpeek/default.nix]: Ah, yes :) Should I force push this change, or add a commit?

Comment (Lassulus) [reply pkgs/applications/audio/sndpeek/default.nix]: force push

**Review by Lassulus — APPROVED** [review]
tested with nix-review, seems to work
